data-processing-lib/doc/advanced-transform-tutorial.md

```diff
@@ -173,15 +170,15 @@ adds their handles to the transform parameters
         )
         return {"hashes": self.filters} | self.params
 ```
-Inputs to this method includes a set of parameters, that moght not be needed for this transformer, but 
+Inputs to this method includes a set of parameters, that moght not be needed for this transformer, but
 rather a superset of all parameters that can be used by different implementations of transform runtime (
-see for example [block listing](../../transforms/universal/blocklist), 
+see for example [block listing](../../transforms/universal/blocklisting),
```

Comment:
this is not in the open source (yet). And it is call `blocklist` not `blocklisting`.

Reply:
fixed

Reply:
fixed